gradle-examples/multi-module/build.gradle

```diff
@@ -16,6 +16,36 @@ buildscript {
 
 String thorntailVersion = System.getProperty('thorntailVersion') ?: VERSION_THORNTAIL
 
+// We are going to publish a "BOM" style artifact from this module.
+apply plugin: 'java-platform'
+
+// ----------------------------------------------------
+//
+// Define the constraints exported by this platform.
+//
+// ----------------------------------------------------
+
```

Comment:
The below section defines the approach that I have been using for publishing BOMs out of a Gradle project. If you have other ideas, I would love to hear about them.